examples/models/llama/source_transformation/quantize.py

```diff
@@ -119,11 +119,10 @@ def quantize(  # noqa C901
         # Check for required args
         if group_size is None:
             raise Exception("For 8da4w quantization, group size must be specified.")
-        from torchao.quantization.quant_api import Int8DynActInt4WeightQuantizer
 
-        model = Int8DynActInt4WeightQuantizer(
-            precision=torch_dtype, groupsize=group_size
-        ).quantize(model)
+        from torchao.quantization import int8_dynamic_activation_int4_weight, quantize_
+
+        quantize_(model, int8_dynamic_activation_int4_weight(group_size=group_size))
```

Comment:
is torch_dtype not applied here? should it be applied to `model`?

Reply:
Yeah I think it would good to ensure the dtype here by applying it to the model in general the model is already in fp32 for when the test passes before this PR and when it fails after this PR